pmg/search.py

```diff
@@ -126,18 +125,43 @@ def mapping(self, data_type):
                     "type": "string",
                     "analyzer": "english",
                     "index_options": "offsets",
+                    "term_vector": "with_positions_offsets",
+                    "store": "yes",
+                    "fields": {
+                        "exact": {
+                            "type": "string",
+                            "analyzer": "english_exact",
+                            "term_vector": "with_positions_offsets",
+                        },
+                    },
                 },
                 "fulltext": {
                     "type": "string",
                     "analyzer": "english",
                     "index_options": "offsets",
                     "term_vector": "with_positions_offsets",
+                    "store": "yes",
```

Comment:
@longhotsummer this is needed to get highlighting on quoted and unquoted with less boilerplate.

Do you have a sense for how much more space this might use? The server has about 2G left so I should probably free some before deploying anyway and monitor as I deploy.

Reply:
I don't know. I'd imagine the current index isn't bigger than a GB or so, but it's a guess.

Reply:
I managed to make a lot of space with a bit of apt and docker cleanup so this is no concern any more.